README.md

```diff
@@ -141,7 +141,7 @@ Additional event properties can be specified as part of `EventModel`
 
 ```kotlin
 val event = Event(EventMetric.VIEWED_PRODUCT)
-    .setProperty(EventKey.VALUE, "10")
+    .setProperty(EventKey.VALUE, 10)
```

Comment:
this is also a breaking change we should write out right?

Reply:
sorry meant this to be for the setValue func

Reply:
Ah yeah, mention it explicitly in the migration guide you mean?

Reply:
yup!

Reply:
Added code sample 👍